images/ose-nutanix-cloud-controller-manager.yml

```
@@ -0,0 +1,19 @@
+content:
```

Cloud-specific images are usually restrained to the arches of that cloud:
```suggestion
arches:
- x86_64
content:
```